static/config/99datadog-windows.config

```
@@ -14,8 +14,8 @@ commands:
     command: start /wait msiexec /qn /i datadog-agent-7-latest.amd64.msi APIKEY="YOUR API KEY" SITE="datadoghq.com"
     cwd: c:/Datadog
 
-  02_setup-APM1:
+  02_setup-APM1: # Stops was for the purpose of setting up tracing on IIS
```

It took me a while to realize this was "was" the Windows Service, not from the verb "to be". 😅 
```suggestion
  02_setup-APM1: # Stops "was" for the purpose of setting up tracing on IIS
```

---

and then the same for "w3svc" below, for consistency